main.cpp

```diff
@@ -52,106 +52,105 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: knife
+//  action 1: Stab a Player
+knife.stab();
+//  action 2: Slash a Player
+knife.slash();
+//  action 3: Hold knife Menacingly
+knife.menace();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: cat
+//  action 1: Make a meow sound
+cat.meow();
+//  action 2: scratch an attacker
+cat.scratchAttacker();
+//  action 3: Jump
+cat.jump();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: box
+//  action 1: Move the box
+box.move();
+//  action 2: Rotate the Box clockwise
+box.rotateClockwise();
+//  action 3: Flip the box
+box.flipX();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: dice
+//  action 1: Roll the dice
+dice.roll();
+//  action 2: pick up the dice
+dice.pickup();
+//  action 3: Drop the Dice
+dice.drop();

```

Comment:
You need to pick actions that the Noun performs.  These action is performed by something or someone other than the Noun ONTO the noun.
- [ ] revise